.github/workflows/README.md

```diff
@@ -0,0 +1,104 @@
+# Stencil Continuous Integration (CI)
```

Comment:
This readme is intentionally high level. I didn't wanna regurgitate the details of every job/step here. If there's something y'all have questions about lmk. An inline comment somewhere may be necessary